src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  onResetSelectedTodo: () => void;
+  selectedTodo: Todo;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  onResetSelectedTodo,
+  selectedTodo,
+}) => {
+  const [loadingTodoModal, setLoadingTodoModal] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setLoadingTodoModal(true);
+    getUser(selectedTodo.userId)
+      .then(setUser)
+      .finally(() => setLoadingTodoModal(false));
+  }, [selectedTodo.userId]);
+
+  const handleResetUserAndTodo = () => {
```

Comment:
The name of this function is not clear. According to the react naming conventions, it should start with 'handle' and then describe what it does. For example, you could rename it to 'handleResetUserAndTodo'.